authmiddleware.js

```diff
@@ -0,0 +1,20 @@
+const express = require('express');
+const jwt = require('express-jwt');
+const jwksRsa = require('jwks-rsa');
+const jwtAuthz = require('express-jwt-authz');
+var request = require('request');
+require('dotenv').config();
+
+module.exports = {
+  verify: jwt({
+    secret: jwksRsa.expressJwtSecret({
+      cache: true,
+      rateLimit: true,
+      jwksRequestsPerMinute: 5,
+      jwksUri: process.env.JWKS_URI
+    }),
+    aud: process.env.AUDIENCE,
+    issuer: process.env.ISSUER,
+    algorithms: ['RS256']
+  })
+};
```

Comment:
does this just return a 401 if the jwt is invalid?

Reply:
500 and a message